clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -922,16 +925,22 @@ static Expected<StringRef> linkDevice(ArrayRef<StringRef> InputFiles,
     SmallVector<OffloadFile> Binaries;
     if (Error Err = extractOffloadBinaries(Buffer->getMemBufferRef(), Binaries))
       return std::move(Err);
+    bool CompatibleBinaryFound = false;
     for (auto &Binary : Binaries) {
       auto BinTriple = Binary.getBinary()->getTriple();
       if (BinTriple == Triple.getTriple()) {
         auto FileNameOrErr = writeOffloadFile(Binary);
         if (!FileNameOrErr)
           return FileNameOrErr.takeError();
         ExtractedDeviceLibFiles.emplace_back(*FileNameOrErr);
+        CompatibleBinaryFound = true;
       }
     }
+    if (!CompatibleBinaryFound)
+      WithColor::warning(errs(), LinkerExecutable)
+          << "Compatible SYCL device library binary not found\n";
```

Comment:
nit.
According to [LLVM style](https://jira.devtools.intel.com/browse/CMPLRLLVM-55044) I would propose the following format:
`not found SYCL device library binary compatible with @Triple`, where `@Triple` is the function argument.